Source/Core/DolphinQt2/GameList/GameListModel.cpp

```diff
@@ -63,7 +65,16 @@ QVariant GameListModel::data(const QModelIndex& index, int role) const
     break;
   case COL_TITLE:
     if (role == Qt::DisplayRole || role == Qt::InitialSortOrderRole)
-      return game->GetLongName();
+    {
+      QString display_name = QString::fromStdString(m_title_database.GetTitleName(
+          game->GetGameID().toStdString(), game->GetPlatform() == tr("Wii Channel") ?
```

Comment:
Comparing against a translated string seems fragile, could you use `GetPlatformID`?